### PR TITLE
[Feature]Load all existing nodes into default warehouse

### PR DESCRIPTION
**Pull request by abc982627271**
Fixes #issue

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/warehouse/LocalWarehouse.java [hunk 1]
```diff
@@ -22,13 +23,22 @@
...
 
 // on-premise
 public class LocalWarehouse extends Warehouse {
     @SerializedName(value = "cluster")
     Cluster cluster;
 
+    public static final ImmutableList<String> CLUSTER_PROC_NODE_TITLE_NAMES = new ImmutableList.Builder<String>()
+            .add("ClusterId")
+            .add("workerGroupId")
+            .add("cnIds")
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/warehouse/LocalWarehouse.java:37]:
```suggestion
            .add("WorkerGroupId")
            .add("ComputeNodeIds")
```

File: fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java [hunk 2]
```diff
@@ -1225,6 +1225,8 @@ private void transferToLeader() {
                 initDefaultWarehouse();
             }
 
+            updateDefaultWarehouse();
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java:1228]: how followers recover default warehouse?

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 23 / 31 (74.19%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/ast/ShowClustersStmt.java|0|2|00.00%|[28, 29]|
|:large_blue_circle:|com/starrocks/warehouse/Cluster.java|2|4|50.00%|[67, 68]|
|:large_blue_circle:|com/starrocks/warehouse/LocalWarehouse.java|7|11|63.64%|[72, 73, 74, 75]|
|:large_blue_circle:|com/starrocks/server/WarehouseManager.java|7|7|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/ShowExecutor.java|4|4|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 19 / 31 (61.29%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/ast/ShowClustersStmt.java|0|2|00.00%|[28, 29]|
|:large_blue_circle:|com/starrocks/qe/ShowExecutor.java|0|1|00.00%|[2460]|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|3|6|50.00%|[1386, 1387, 1390]|
|:large_blue_circle:|com/starrocks/warehouse/Cluster.java|2|4|50.00%|[67, 68]|
|:large_blue_circle:|com/starrocks/warehouse/LocalWarehouse.java|7|11|63.64%|[72, 73, 74, 75]|
|:large_blue_circle:|com/starrocks/server/WarehouseManager.java|7|7|100.00%|[]|

**Review by kevincai — APPROVED** [review]

**Review by wyb — APPROVED** [review]

**Review by sduzh — APPROVED** [review]

**Pull request merged**